src/core/Akka/Actor/ActorRefFactoryExtensions.cs

```diff
@@ -19,9 +19,10 @@ public static class ActorRefFactoryExtensions
         /// <param name="factory">TBD</param>
         /// <param name="name">TBD</param>
         /// <returns>TBD</returns>
-        public static IActorRef ActorOf<TActor>(this IActorRefFactory factory, string name = null) where TActor : ActorBase, new()
+        public static IActorRef ActorOf<TActor>(this IActorRefFactory factory, string name = null)
+            where TActor : ActorBase, new()
         {
-            return factory.ActorOf(Props.Create<TActor>(), name: name);
+            return factory.ActorOf(Props.Create<TActor>(), name);
```

Comment:
removed named parameter since the method only takes two parameters and its obvious which parameter is which